roles/common/templates/etc/serverspec/kvm_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'spec_helper'
+
+describe file('/var/lib/libvirt/images') do
+  it { should be_mode 711 }
+end
+
+describe file('/dev/kvm') do
+  it { should exist }
+  it { should be_mode 660 }
+end 
+
+describe file('/etc/libvirt/libvirtd.conf') do
+  file_contents = ['unix_sock_group = "libvirtd"',
+                   'unix_sock_ro_perms = "0777"',
+                   'unix_sock_rw_perms = "0770"']
+  file_contents.each do |file_line|
+    it { should contain file_line }
+  end
+end
+
+describe file('/etc/libvirt/libvirtd.conf') do
+  file_contents = ['#listen_tcp = 1',
+                   'listen_tcp = 1',
+                   '#listen_tls = 0',
+                   'listen_tls = 0']
+  file_contents.each do |file_line|
+    it { should contain file_line }
+  end
+end
+
+describe command('cat /etc/libvirt/libvirtd.conf|grep ^listen_') do
+  file_contents =["listen_tls = 0", "\nlisten_tcp = 1\n"].join()
+  its(:stdout) { should eq file_contents }
+end
```

Comment:
Isn't this check the same as the check above it? Why are you checking for listen_tls and listen_tcp twice?